Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.16-alpine AS base
```

Comment:
Same thing here. Are these (Dockerfile and docker-compose.yaml) just for Air? Do you envision them being in the main repo?

Reply:
i made these just in case someone's configurations would use these, as I was using a boilerplate tutorial that supplied them.